src/compiler/scala/tools/nsc/settings/Warnings.scala

```diff
@@ -26,7 +26,12 @@ trait Warnings {
   // Warning semantics.
   val fatalWarnings = BooleanSetting("-Werror", "Fail the compilation if there are any warnings.") withAbbreviation "-Xfatal-warnings"
 
-  private val WconfDefault = List("cat=deprecation:ws", "cat=feature:ws", "cat=optimizer:ws")
+  private val WconfDefault = List(
+    "cat=deprecation:ws",
+    "cat=deprecation-error:error",
```

Comment:
```suggestion
    "cat=deprecation-error:e",
```

Reply:
I used the longer option intentionally for the purpose of readability. I think I came across the idea here https://kvz.io/bash-best-practices.html a while back:

> Use long options (logger --priority vs logger -p). If you're on cli, abbreviations make sense for efficiency. but when you're writing reusable scripts a few extra keystrokes will pay off in readability and avoid ventures into man pages in the future by you or your collaborators.